src/proc/spatial-filter.cpp

```diff
@@ -210,4 +221,265 @@ namespace librealsense
         memmove(const_cast<void*>(tgt.get_data()), f.get_data(), _current_frm_size_pixels * _bpp);
         return tgt;
     }
+
+
+
+    void spatial_filter::recursive_filter_horizontal_fp(void * image_data, float alpha, float deltaZ)
```

Comment:
Is the code below being called or is it just for reference?

Reply:
The section is functional - a copy of the reference code to be invoked for floating-point data filtering